src/components/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ buttonName }) => (
+  <div>
+    <button type="button">
+      { buttonName }
+    </button>
+  </div>
+);
+
+Button.propTypes = {
+  buttonName: PropTypes.string,
+};
+
+Button.defaultProps = {
+  buttonName: '',
+};
```

Comment:
A `buttonName` can't be empty since each button will always have a name.
```suggestion
Button.propTypes = {
  buttonName: PropTypes.string.isRequired,
};

```